ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -313,6 +313,20 @@ <h3>Source database details:</h3>
         Note: Spanner migration tool is creating datastream and dataflow resources. Please look at the terminal logs to
         check the progress of resource creation. All created resources will be displayed here once they are generated.
     </div>
+    <div *ngIf="isResourceGenerated && resourcesGenerated.MonitoringDashboardName!==''">
+      <h3> Monitoring Dashboards:</h3>
+      <span *ngIf="resourcesGenerated.MonitoringDashboardName!=='' && (selectedMigrationType ==='lowdt') && !isSharded">
```

Comment:
This condition seems to be repetitive since we have already checked in line 316 : resourcesGenerated.MonitoringDashboardName!==''